test/test_distributed.py

```diff
@@ -1656,6 +1656,9 @@ def _test_DistributedDataParallel(self, gpu_subset, rank, output_device=None):
         # Run a simple end to end DDP model, use result of single node model
         # as baseline
 
+        # a module without gradients shouldn't be accepted
+        self.assertRaises(AssertionError, lambda: nn.parallel.DistributedDataParallel(nn.Module(), device_ids=gpu_subset))
+
```

Comment:
Can you turn this into its own test case? This function is just a runner, and there is no need to re-check the assertion from every test.

Reply:
Updated thank you :)